### PR TITLE
Move email settings to config.php

### PR DESCRIPTION
**Pull request by Derkades**
This will also ease migration to [dotenv](https://github.com/vlucas/phpdotenv) later, we can just make the Config class return values from env instead

### CODE REVIEW

**Comment by Derkades** [comment]
Oops, I thought Config::get had a second parameter for default value..

**Comment by Derkades** [comment]
Migration has been tested

**Review by Aberdeener — APPROVED** [review]
Thank you 🤟🏼

**Comment by Aberdeener** [comment]
Before:
Screen Shot 2022-06-21 at 22 39 36

After:
Screen Shot 2022-06-21 at 22 39 28

Sooo much cleaner, kept forgetting to do this.

Side-bar, @Derkades why is `trustedProxies` in camel case? Can we make it snake case for consistency?

File: core/migrations/20220620190645_move_email_settings_to_config.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    public function change(): void {
+        // New installations won't have the old email.php config file
+        if (!is_file(ROOT_PATH . '/core/email.php')) {
+            return;
+        }
+
+        // Migrate settings from core/email.php to core/config.php
+        require(ROOT_PATH . '/core/email.php');
+        Config::set('email.email', $GLOBALS['email']['email']);
```
[the remark this answers is not in the document]
Comment (Aberdeener) [reply core/migrations/20220620190645_move_email_settings_to_config.php]: Changed

**Comment by Derkades** [comment]
The reason why `trustedProxies` is camel case is because during the creation of it I removed the old `allowedProxies` option. The naming of that option made me think camcel case was the convention without looking at other options

**Pull request merged**